dxr/menus.py

```diff
@@ -0,0 +1,12 @@
+"""Common menu constructors for use in plugins."""
+from os.path import basename
+
+from dxr.utils import browse_url
```

Comment:
I'm afraid we're going to have to rebase this one (or at least merge in from es); browse_url doesn't exist anymore.